packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1781,6 +1794,10 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
   }
 
   private _onOptionMouseMove(index: number): void {
+    if (!this._gotMouseMove) {
```

Comment:
> if (!this._gotMouseMove) { [](start = 3, length = 27)

[nit] [nit] I think there is almost no perf difference that might as well just have this._gotMouseMove = true; here and get rid of the if check